stingray/io.py

```diff
@@ -802,16 +802,37 @@ def _isattribute(data):
     ----------
     data : object
         The object to be checked.
+        The object must be list, list of tuples, tuple, tuple of tuples, tuple of lists and ndarrays.
 
     Returns:
         bool
         True if the data is a single number, False if it is an iterable.
-    """
+    
+    Examples
+    ----------
+    >>> _isattribute(1)
+    True   
+    >>> _isattribute([1])
+    True
+    >>> _isattribute((1,2))
+    False
+    >>> _isattribute([1,2])
+    False
+    >>> _isattribute([1,2,3])
+    False
+    
+    some more subtle examples:
+    >>> _isattribute([[1]]))
```

Comment:
Syntax errors here. Check double brackets! The test failure comes from here.

Reply:
I have corrected them.